src/commands.js

```diff
@@ -0,0 +1,47 @@
+const { bookshelf,
+  Train,
+  Trains,
+  Passenger,
+  Passengers,
+  Station,
+  Stations,
+  Ticket,
+  Tickets,
+  StationAndTrain,
+  StationAndTrainRecords,
+  StationAndPassenger,
+  StationAndPassengerRecords,
+  TrainAndPassenger,
+  TrainAndPassengerRecords
+} = require('../database/connection.js')
+
+const retrieveRecord = ( model ) => {
+  return model.fetch()
+        .then( result => {
```

Comment:
I advise against indentation in this manner; it becomes a hassle when maintaining code.  A simple two space indent, consistently applied throughout your codebase, is sufficient to indicate chaining with the previous line.  This formatting feels odd to me as well because I don't typically see chained code left aligned with the first function invoked in the chain.